anycloud/node/src/datastore.ts

```diff
@@ -1,67 +1,77 @@
-require('cross-fetch/polyfill')
+import fetch from 'cross-fetch';
+const https = require('https');
 
-class DataStore {
+export class DataStore {
   private localDS: any;
-  private ns: string;
   private clusterSecret?: string;
   private isLocal: boolean;
   private headers?: any;
   private ctrlPortUrl: string;
 
   constructor() {
     this.localDS = {};
-    this.ns = 'kv';
     this.clusterSecret = process.env.CLUSTER_SECRET;
     this.isLocal = this.clusterSecret ? false : true;
-    this.headers = this.clusterSecret ? {[this.clusterSecret]: 'true'} : null;
-    this.ctrlPortUrl = 'https://localhost:4142';
+    this.headers = this.clusterSecret ? { [this.clusterSecret]: 'true' } : null;
+    this.ctrlPortUrl = 'https://localhost:4142/app/';
+    // To avoid failure due to self signed certs
+    https.globalAgent.options.rejectUnauthorized = false;
```

Comment:
We are gonna want to constrain that to our library and not mutate the global agent or this library becomes a security vulnerability. We can do this only because we are directly connecting to localhost, but this change would affect all domains.

Reply:
Yes, I was going to ask about this. This change the global request agent and that's not ok, but the `cross-fetch` package does not support pass a custom `agent`. I've seen [`node-fetch`](https://www.npmjs.com/package/node-fetch) and apparently we can instantiate a new agent and pass it through the fetch. Should we change library?

Reply:
Yeah; thinking about it this library is never going to be used in a browser context so polyfilling isn't necessary.